crates/core/tedge_mapper/Cargo.toml

```diff
@@ -38,6 +38,9 @@ tedge_uploader_ext = { workspace = true }
 tracing = { workspace = true }
 
 [features]
+default = ["aws"]
```

Comment:
I'd either drop that default or define `c8y` as the default, as that's the most commonly used one (provided that `c8y_mapper_ext` is also put behind a feature flag).

Reply:
Okay, just saw [this comment](https://github.com/thin-edge/thin-edge.io/pull/3585#issuecomment-2835473791)